src/containers/ValidatorDetails/components/Details/components/ValidatorInfo/index.tsx

```diff
@@ -29,6 +29,7 @@ import useDelegationModal from 'components/Dialog/components/DelegationModal/hoo
 import { ModalStatus } from 'store/modal'
 import { OverviewType, StatusType } from '../../types'
 import { getValidatorStatus, getCondition } from './utils'
+import useValidators from '../../../../../Staking/components/Validators/components/Table/hooks'
```

Comment:
can be imported just from "components/Validators...etc"

Reply:
Not applicable in this case as they are in different "components" folders